### ISSUE TITLE
previousFlags private property?

### ISSUE THREAD

**Issue by karadis** [ashleymills/Reachability.swift#78]
Hi, how can i know previous states of reachability (without saving it locally) in reachabilityChanged notification handler?
'previousFlags' property is private even though its marked under public properties (mistake?)

**Comment by ashleymills** [comment]
Yeah - that was a mistake. Currently you need to save the state locally. I'm considering updating the `whenReachable`  closure signature to something like:

`(status: NetworkStatus, previousStatus: NetworkStatus, reachability:Reachability) -> ()`

I'm guessing that would solve your problem?
